p5/sketch/Vispy2DRenderer/renderer2d.py

```diff
@@ -375,3 +376,52 @@ def arc(self, *args):
     def shape(self, vertices, contours, shape_type, *args):
         """Render a Pshape"""
         self.render_shape(PShape(vertices=vertices, contours=contours, shape_type=shape_type))
+
+    def reset_transforms(self):
```

Comment:
Looks like this (along with a few other functions) is duplicated between vispy2d and 3d as well. Could you also move them to the common opengl renderer?

Reply:
Moved the functions, and declared an abstract `render` function in `openglrenderer` for `shape`